concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme.md

```diff
@@ -12,14 +12,30 @@ Configure the module bundler you are using and import `dx.common.css` and a them
 2. Open the DevExtremeBundleConfig.cs file
 3. Find the following code and replace `dx.light.css` in it with the theme stylesheet of your choice:
 
+            <!-- tab: DevExtremeBundleConfig.cs -->
             styleBundle
                 .Include("~/Content/dx.common.css")
                 .Include("~/Content/dx.light.css");
 
+4. Open the Views/Shared/_Layout.cshtml file and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
+
+            <!-- tab: _Layout.cshtml -->
+            <body style="padding-top: 5rem;" class="dx-viewport">
+                <!-- ... -->
+            </body>
+
 - **ASP.NET Core**      
 
 1. Go to the Views/Shared folder (for conventional Razor views) or Pages folder (for Razor Pages).
 2. Open the _Layout.cshtml or _DevExtremeLayout.cshtml file (depending on which of them is used in the _ViewStart.cshtml file).
```

Comment:
Change to:
2. Open the _Layout.cshtml or _DevExtremeLayout.cshtml file (depending on which file is used in _ViewStart.cshtml).

I used the word "file", but since you have a better understanding of the topic, if there is a more precise word, please feel free to suggest it.